third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -28,7 +28,6 @@ func TestAccContainerNodePool_basic(t *testing.T) {
 				ResourceName:      "google_container_node_pool.np",
 				ImportState:       true,
 				ImportStateVerify: true,
-				ImportStateVerifyIgnore: []string{"max_pods_per_node"},
```

Comment:
Good catch on these- it looks like there are a bunch more, do you mind getting them too?

Reply:
Done.